engine/src/flutter/shell/platform/android/platform_view_android.cc

```diff
@@ -106,15 +106,14 @@ PlatformViewAndroid::PlatformViewAndroid(
     PlatformView::Delegate& delegate,
     const flutter::TaskRunners& task_runners,
     const std::shared_ptr<PlatformViewAndroidJNI>& jni_facade,
-    bool use_software_rendering)
+    AndroidRenderingAPI rendering_api)
```

Comment:
Wait, if you don't store this - how is it getting here?

Reply:
Based on the failing test... it does not 😅.

Reply:
I think it has to go back to settings unfortunately. I may have run into the same problem that there is no where else to stash it.

Reply:
Uploaded an _option_.

It's not great, but it's not different than storing (and mutating) `Settings`.

More changes (potentially in follow-up PRs) could make this cleaner. Wdut?